backend/tracim_backend/command/__init__.py

```diff
@@ -63,6 +63,9 @@ def take_action(self, parsed_args: argparse.Namespace) -> None:
                         self.take_app_action(parsed_args, app_context)
         except Exception as exc:
             logger.exception(self, exc)
+            import traceback
```

Comment:
logger.exception(self, exc) is not enought to get traceback ?

Reply:
the logs do not appear in the tests, that's why I added this and I forgot it. I didn't check if logs are displayed while using `tracimcli` interactively.